### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by NouraAlhadlaq**
How difficult did you find this (out of 10)?
 0 
Was there anything that you struggled with?
no there wasn't.
Is there anything that you'd like some further information on?;
Roughly how long did it take?
1 hour

### CODE REVIEW

File: Noura_Alhadlaq/week_01/day_02/main.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+var currentTemp = 40;
+var acFunctional = false;
+var desiredTemp = 30;
+
+if(acFunctional && currentTemp > desiredTemp){
+	console.log("Turn on the A/C Please");
+} else if(! acFunctional){
+	if(currentTemp > desiredTemp){
+	console.log("Fix the A/C now! It's hot!");
+} else if(currentTemp < desiredTemp){
```
Comment (mfalthaw) [inline Noura_Alhadlaq/week_01/day_02/main.js]: Good work here! Something to think about; what will be printed if the `AC is functional` AND `the current temp is less than the desired temp`

File: Noura_Alhadlaq/week_01/day_02/main.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+} else if (score === par){
+	nickname = "Par";
+	console.log(nickname);
+} else if (score === par+1){
+	nickname = "Bogey";
+	console.log(nickname);
+} else if (score === par+2){
+	nickname = "Double Bogey";
+	console.log(nickname);
+} else if (score === par+3){
```
Comment (mfalthaw) [inline Noura_Alhadlaq/week_01/day_02/main.js]: Great work! What about when the score is `> par + 3`?

File: Noura_Alhadlaq/week_01/day_02/main.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+var message = "   ";
+
+//import isUpperCase from 'is-upper-case';
+
+if(message.endsWith("?")){
+	console.log("Sure");
+} //else if (){
+	//console.log("Waoh, chill out!");} 
+else if (message.length === 0 || /^\s*$/.test(message)){
```
Comment (mfalthaw) [inline Noura_Alhadlaq/week_01/day_02/main.js]: Nice use of regular expressions! regex can also be used to check for _ALL CAPS_ as well.

**Comment by mfalthaw** [comment]
Great work on your first homework @NouraAlhadlaq ! I left some comments for you.

File: Noura_Alhadlaq/week_01/day_02/main.js (hunk 2 above)
Comment (NouraAlhadlaq) [reply Noura_Alhadlaq/week_01/day_02/main.js]: add this to the end :
else {
	console.log(nickname);
}

**Pull request merged**